usagov/usagov-2021
# USAGOV-1911: nav-page-spacing-update: update for feedback on nav page

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1911

## Description

Update to reduce spacing above and below the breadcrumb on nav pages
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
This is more of a visual test to see if the spacing above and below the breadcrumb.
Goto http://localhost/about-the-us on desktop and then view the spacing. The spacing should look the same in  tablet and mobile.

<img width="1512" alt="Screenshot 2024-09-04 at 7 18 21 AM" src="https://github.com/user-attachments/assets/af166d1c-cbb7-46b7-8d2a-e8646cef3b50">



<img width="1512" alt="Screenshot 2024-09-04 at 7 19 00 AM" src="https://github.com/user-attachments/assets/c2343f84-b76a-47ea-abc2-f73979e0ffc4">


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
